app/dashboard/migrations/0173_merge_20210223_1145.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.4 on 2021-02-23 11:45
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboard', '0172_auto_20210216_0820'),
+        ('dashboard', '0169_merge_20210112_1159'),
```

Comment:
this one too

Reply:
You mean I delete those two migrate files?